### PR TITLE
[ExtensionCard] UI Update (10/1)

### PR DESCRIPTION
**Pull request by msangapu-msft**

### CODE REVIEW

**Comment by msangapu-msft** [comment]
image

I think it could use some spacing between cards. Any feedback on styling?

**Comment by craigshoemaker** [comment]
Looking good!

Certainly need bottom margin between cards. The title link needs to be colored blue to indicate you can click on it better. Look to make changes in the *app.css* file associated with the `card` class so that repository cards get the same treatment.

As I work with the extension data service I'll see what other data we can feed into this card.

File: ui/src/components/ExtensionCard.vue [hunk 1]
```diff
@@ -2,18 +2,10 @@
...
       <div>
-        <span class="font-semibold">Timestamp:</span>
-        <span class="extension-tag">{{ resource.Timestamp }}</span>
-      </div>
-      <div>
-        <span class="font-semibold">URL:</span>
-        <a :href="resource.url"
-          ><span class="extension-tag">{{ resource.url }}</span></a
-        >
+        <span class="extension-tag">{{ resource.description }}</span>
```
Comment (craigshoemaker) [inline ui/src/components/ExtensionCard.vue]:
```suggestion
        <p>{{ resource.description }}</p>

```

The `extension-tag` class is used to style the metadata tags for a resource.

**Comment by msangapu-msft** [comment]
Here's the latest render
image

**Comment by msangapu-msft** [comment]
@craigshoemaker Let me know if this needs any more changes.

**Pull request merged**